endpoints/openrtb2/auction.go

```diff
@@ -480,6 +484,10 @@ func validateNativeContextTypes(cType native.ContextType, cSubtype native.Contex
 }
 
 func validateNativePlacementType(pt native.PlacementType, impIndex int) error {
+	if pt == 0 {
```

Comment:
Can we please add a unit test or json test for this case?

Reply:
See the other JSON tests created/moved to reflect the new rules.

Reply:
Ah. Changing the directory changes the validation criteria. Missed that.